.github/workflows/integration_tests.yml

```diff
@@ -528,6 +528,56 @@ jobs:
           fi
         shell: bash
 
+      - name: Submit to Fermioniq test server
+        if: (success() || failure()) && (inputs.target == 'fermioniq' || github.event_name == 'schedule' || inputs.target == 'nightly')
```

Comment:
Thanks @MarkusPfundstein.
This looks good. Since `inputs.target` is a `choice` option, we'd need to add `fermioniq` to the list (line 25 of this file) as well.

Reply:
Ok will done. Thx